src/bookmarklet/inject_base.js

```diff
@@ -193,11 +193,15 @@ translate.setHandler("attachmentProgress", function(obj, attachment, progress) {
 translate.setHandler("done", function(obj, returnValue) {
 	if(returnValue && haveItem) {
 		Zotero.ProgressWindow.startCloseTimer(2500);
+		cleanup();
 	} else if(!cancelled) {
-		new Zotero.ProgressWindow.ErrorMessage("translationError");
-		Zotero.ProgressWindow.startCloseTimer(8000);
+		Zotero.ProgressWindow.changeHeadline("We could not extract the metadata.");
```

Comment:
looks like this message won't be translated, while the rest of STR does support different languages.

Reply:
Can you point me to a PME text that gets translated?
This is a Zotre progress window headline.

Reply:
Example: The current error that happened was a PME error. Should not have been translated from what I know.

Reply:
You are right, PME != STR. STR is the one that has translations.